controllers/operands/kubevirt_test.go

```diff
@@ -3015,14 +3015,14 @@ Version: 1.2.3`)
 					Expect(kv.Spec.Workloads).To(BeNil())
 				})
 
-				It("should not set replica with SingleReplica ControlPlane but HighAvailable Infrastructure ", func() {
+				It("should set replica=1 with SingleReplica ControlPlane and HighAvailable Infrastructure ", func() {
 					hcoutil.GetClusterInfo = func() hcoutil.ClusterInfo {
 						return &commontestutils.ClusterInfoSRCPHAIMock{}
 					}
 					kv, err := NewKubeVirt(hco)
 					Expect(err).ToNot(HaveOccurred())
 					Expect(kv.Spec.Infra).To(Not(BeNil()))
-					Expect(kv.Spec.Infra.Replicas).To(BeNil())
+					Expect(*kv.Spec.Infra.Replicas).To(Equal(uint8(1)))
```

Comment:
nit - prefer using the `HaveValue` matcher, over getting the pointer value directly, because this is also check the `!= nil` case.